packages/patternfly-3/patternfly-react/src/components/DualListSelector/components/DualListHeading.js

```diff
@@ -20,14 +20,12 @@ const DualListHeading = ({
   kebabID
 }) => (
   <div className="dual-list-pf-heading">
-    <Form inline>
```

Comment:
can't have a nested form if the DualList will be wrapped in a form.

Reply:
+1, imho this component should never introduce a form by it self, the same goes for hidden value, that would be up to developer to decide (out of context of patternfly).

Reply:
I think many consumers would be happy to receive this component as inputs to be able to submit it instead of connecting to it which might be more complex.
we can add a prop for it to let the consumers decide by themselves.

@jeff-phillips-18 @sharvit @glekner what are your thoughts about this?